packages/frint/README.md

```diff
@@ -304,6 +304,8 @@ The `App` instance
 
 `app.getOption('name')` would give you `MyAppName` string.
 
+or better `app.getName()` Since name in options is a required key.
```

Comment:
please also add a dedicated heading for `app.getName`. it can go after `app.getOption`: https://github.com/Travix-International/frint/blob/master/packages/frint/README.md#app-1

```md
### app.getName

> app.getName()

#### Returns

`String`: The App's name.
```